postgres/changelog.d/17144.added

```diff
@@ -0,0 +1 @@
+add wal metrics
```

Comment:
This changelog message is pretty vague, can you please add some additional context here? customers will look at this